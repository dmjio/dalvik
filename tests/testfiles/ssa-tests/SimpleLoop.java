class SimpleLoop {
  static int entry(int x) {
    int z = 0;
    for(int i = 0; i < x; ++i) {
      z += x;
    }

    return z;
  }
}
