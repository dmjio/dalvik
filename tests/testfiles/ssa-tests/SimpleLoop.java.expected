SimplePhi { numBlocks = 4
          , phis = [ (0, fromList ["0", "%19"])
                   , (1, fromList ["0", "%20"])
                   ]
          , blockPhis = [(0, 1), (1, 1)]
          , assignedPhis = []
          }